### PR TITLE
Jetpack Cloud: Add header and paragraph content for the Scan Threats page

### PR DESCRIPTION
**Pull request by robertf4**
#### Changes proposed in this Pull Request

This adds the header and first paragraph content to the Scan Threats page.

Overall design:
image

Part added by this PR:
image

Note that the design shows the number of threats being spelled out, i.e. "two" instead of "2". Due to the technical difficulties of doing that in an I18N environment the number value is displayed instead.

#### Testing instructions
1. Checkout the branch and start Calypso with `CALYPSO_ENV=jetpack-cloud-development npm start`.
2. Visit `http://calypso.localhost:3000/scan/$site_slug?scan-state=threats`.
3. Verify that the design of the header and paragraph content matches the Figma file.

### CODE REVIEW

File: client/landing/jetpack-cloud/sections/scan/main.jsx [hunk 1]
```diff
@@ -49,7 +49,35 @@ class ScanPage extends Component {
...
+					{ translate(
+						'The scan found {{strong}}%(threatCount)s{{/strong}} potential threats with {{strong}}%(siteName)s{{/strong}}.',
+						{
+							args: {
+								siteName: site.name,
+								threatCount: numberFormat( threats.length ),
+							},
+							components: { strong: <strong /> },
+							comment:
+								'%(threatCount)s represents the number of threats currently identified on the site, and $(siteName)s is the name of the site.',
```
Comment (rcanepa) [inline client/landing/jetpack-cloud/sections/scan/main.jsx]: What is the purpose of the comment? I read the `i18n-calypso` README and I didn't find any good explanation of it.

Comment (robertf4) [reply client/landing/jetpack-cloud/sections/scan/main.jsx]: @rcanepa the translators read it for context. It's hard to consistently write good comments but I try to imagine that I'm a translator translating string after string with minimal context and what I would want to know in that situation. So if the string isn't clear out of context, or if there are variables like this, or anything like that then it is a good idea to leave a comment.

**Comment by robertf4** [comment]
> This is not really important, but I was thinking about what you said about displaying the count in words instead of numbers, and I wondered if the idea is to display any number in words or just the first 3 or 5 or 10? For example, how should we display 13 threats? As 13 threats or thirteen threats?

Based on how I read the design, it would be "thirteen threats". I think this is a more "proper" way of writing it. But given that there isn't a function (that I know of) that will do this for all supported languages I think it's best to just go with the number.

File: client/landing/jetpack-cloud/sections/scan/main.jsx (hunk 1 above)
Comment (rcanepa) [reply client/landing/jetpack-cloud/sections/scan/main.jsx]: Oh, I didn't know about the translators. Good to know!

**Pull request merged**